pytorch/inference/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -8,7 +8,8 @@ ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.c
 ARG TORCHVISION_VERSION=0.14.1+cpu
 ARG TORCHAUDIO_VERSION=0.13.1+cpu
 ARG TORCHDATA_VERSION=0.5.1
-ARG TORCHSERVE_VERSION=0.7.0
+ARG TORCHSERVE_VERSION=0.7.1
+ARG SM_TOOLKIT_VERSION=2.0.14
```

Comment:
Should this be 2.0.12? 

cc @lxning to review TS/Toolkit versions

Reply:
yes, tookkit 2.0.14 fix a bug from 2.0.12 telmetry log4j.